students/hyeonjin/users/views.py

```diff
@@ -33,10 +35,13 @@ def post(self, request):
             if User.objects.filter(email=email).exists():
                 return JsonResponse({'message': 'E-mail Duplicate'}, status=400)
 
+            encoded_password = password.encode('utf-8')
+            hashed_password  = bcrypt.hashpw(encoded_password, bcrypt.gensalt())
```

Comment:
```suggestion
            hashed_password  = bcrypt.hashpw(encoded_password, bcrypt.gensalt()).decode('utf-8')
```
데이터베이스에 str타입으로 넣어주기 위해 decode를 해야합니다~

Reply:
오 감사합니다🍀